unstructured/ingest/registry.py

```diff
@@ -0,0 +1,68 @@
+import json
```

Comment:
Does it make sense for this to live inside the connector package? Possibly in the `__init__` file? This is very specific to the connectors rather than the entire ingest code.

Reply:
yep, good point! moving it now.